samples/getting_started/RandomBit.qs

```diff
@@ -0,0 +1,20 @@
+/// # Summary
+/// Random Bit sample
+///
+/// # Description
+/// This Q# program generates a random bit by setting a qubit in a superposition
+/// of the computational basis states |0〉 and |1〉, and returning the measurement
+/// result.
+operation Main() : Result {
+    // Allocate a qubit.
+    use qubit = Qubit();
+
+    // Set the qubit in superposition by applying a Hadamard transformation.
+    H(qubit);
+
+    // Measure then reset the qubit.
+    // There is a 50% probability of measuring either `Zero` or `One`.
+    let result = MResetZ(qubit);
+
+    result
```

Comment:
nitpick: I'm not sure there is value in binding the output just to do an implicit return here. I think for this kind of sample I'd prefer either an implicit return with just the call, as in `MResetZ(qubit)`, or an explicit return with just the call, as in `return MResetZ(qubit);`